### PR TITLE
Rainbow text

### PR DESCRIPTION
**Pull request by IsraelCA**
Please let the invites in
Please
Please
Let the magic begin

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -125,6 +132,11 @@ <h2 class="section-header">Us!</h2>
                     -->
                     
                     <!-- Person #6 write your code between here-->
+                    <img src:"https://www.google.com/search?q=google&source=lnms&tbm=isch&sa=X&ved=0ahUKEwiA-piczvjMAhWn6IMKHVPzBjMQ_AUICigE&biw=1280&bih=647#imgrc=d17bslxLqp9bOM%3A">
```
Comment (lisunshiny) [inline index.html]: should be `src=`

File: index.html [hunk 2]
```diff
@@ -8,7 +8,14 @@
     <body>
         <header>
             <h1>uber advanced class!!!</h1>
-            
+            <style>
+                #Notacountry {
```
Comment (lisunshiny) [inline index.html]: please put this in `styles.css` instead of in its own `style` tag

File: index.html (hunk 1 above)
Comment (lisunshiny) [inline index.html]: wrap everything in an `li` element with class `clearfix`, and everything but the image in a div with class `person-content`

**Comment by lisunshiny** [comment]
done looking

**Comment by IsraelCA** [comment]
Done

File: index.html [hunk 3]
```diff
@@ -125,7 +132,15 @@ <h2 class="section-header">Us!</h2>
                     -->
                     
                     <!-- Person #6 write your code between here-->
-                    
+                    <li class="clearfix"> 
+                     
+                      <img src:"https://pbs.twimg.com/profile_images/638751551457103872/KN-NzuRl.png" width=100px height=130px>
```
Comment (lisunshiny) [inline index.html]: you need a `=` not the `:` after the `src`

File: index.html (hunk 2 above)
Comment (lisunshiny) [inline index.html]: ^dis too

**Pull request merged**